blueprints/screening_app/domains/prediabetes_screening/answer_yes_active/train.txt

```diff
@@ -0,0 +1,102 @@
+hago mucho deporte
```

Comment:
@ritvikshrivastava is there a way to incorporate this kind of flexibility into the slot filling? the answer_yes_* and answer_no_* intents are pragmatically natural responses to the yes/no questions, but don't explicitly say yes or no. adding these phrases into an entity map would work, it seems, but would be more brittle.

Reply:
@serapio They aren't being used now, right? Slot-filling essentially implies getting rid of the binary part and asking for the value outright?

Reply:
One way would be to define a custom validation function and checking against these lists of queries for each of the corresponding slots